README.md

```diff
@@ -1254,6 +1260,80 @@ Celery is a complex topic, so it's a good idea to invest time reading the docume
 
 We constantly do that & find new things or find better approaches to our problems.
 
+## Documentation
```

Comment:
Lets name that `API Documentation`, since `Documentation` may be a broader subject, including texts & diagrams around the project itself :thinking:

Reply:
And then, we can move the section to be below `APIs & Serializers`